video_xblock/static/css/studio-edit.css

```diff
@@ -167,12 +167,12 @@
     transform: translateX(23px);
 }
 
-.default-transcripts-wrapper {
-    display: none;
-}
-.default-transcripts-switch-input:checked ~ .default-transcripts-wrapper {
+.default-transcripts-wrapper:checked {
     display: block;
 }
+.default-transcripts-switch-input:not(:checked) ~ .default-transcripts-wrapper {
```

Comment:
It serves to affect default subs display (show/hide `.default-transcripts-wrapper`) on toggle-switch click (`.default-transcripts-switch-input`).